package.json

```diff
@@ -19,7 +19,7 @@
     "copy:assets": "rsync -rR lib/routes/graphql/schema.gql ./locales ./tsconfig.json ./dist",
     "clean": "rm -rf dist",
     "document": "typedoc --includeVersion",
-    "doctoc": "doctoc README.md",
+    "doctoc": "npx doctoc README.md",
```

Comment:
So always automatically install the latest ver. of doctoc and execute it instead of running a locally installed version?

Reply:
just to remove an unnecessary dependency which had vulnerabilities in the audit